### PR TITLE
Fix subscription schedule

### PR DESCRIPTION
**Pull request by RobWu**

### CODE REVIEW

File: db/migrate/20200520152604_change_stripe_subscription_schedule_phase_id_to_bigint.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class ChangeStripeSubscriptionSchedulePhaseIdToBigint < ActiveRecord::Migration[6.0]
+  def change
+    if postgres?
+      change_column(
+        :stripe_subscription_schedule_phase_plans,
+        :stripe_subscription_schedule_phase_id,
+        "bigint USING stripe_subscription_schedule_phase_id::bigint"
```
Comment (kaspernj) [inline db/migrate/20200520152604_change_stripe_subscription_schedule_phase_id_to_bigint.rb:7]: Why is it necessary with something special for postgres?

File: lib/stripe_model_callbacks/models/stripe_subscription_schedule.rb [hunk 2]
```diff
@@ -35,6 +38,17 @@ def assign_from_stripe(object)
...
 
+  def cancel_on_stripe
+    to_stripe.cancel
+    update!(canceled_at: Time.zone.now) if respond_to?(:canceled_at)
+    reload_from_stripe!
+    true
+  end
+
+  def cancel_on_stripe!
+    raise ActiveRecord::RecordInvalid, self unless cancel_on_stripe
```
Comment (kaspernj) [inline lib/stripe_model_callbacks/models/stripe_subscription_schedule.rb:49]: Can `cancel_on_stripe` return something else than true og raise an error? :-)

File: db/migrate/20200520152604_change_stripe_subscription_schedule_phase_id_to_bigint.rb (hunk 1 above)
Comment (RobWu) [reply db/migrate/20200520152604_change_stripe_subscription_schedule_phase_id_to_bigint.rb:7]: > **SET DATA TYPE**
> ... A USING clause must be provided if there is no implicit or assignment cast from old to new type.

https://www.postgresql.org/docs/9.1/sql-altertable.html

File: lib/stripe_model_callbacks/models/stripe_subscription_schedule.rb (hunk 2 above)
Comment (RobWu) [reply lib/stripe_model_callbacks/models/stripe_subscription_schedule.rb:49]: Used same pattern as in https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb

https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb#L46-L48

https://github.com/kaspernj/stripe_model_callbacks/blob/master/app/models/stripe_model_callbacks/application_record.rb#L57-L59

**Comment by kaspernj** [comment]
Great work :-)

**Pull request merged**